tests/unit/conftest.py

```diff
@@ -25,5 +40,55 @@ def get_auth_token():
     mocker.patch.object(docarray.DocumentArray, 'push', return_args)
     hubble.login()
     client = FinetunerV1Client()
-    mocker.patch.object(client, 'hubble_user_id', '1')
+    mocker.patch.object(client, 'hubble_user_id', HUBBLE_USER_TEST_ID)
     return client
+
+
+@pytest.fixture
+def test_finetuner(test_client, mocker):
```

Comment:
I would define the mock objects in a separate `mocks` directory. These aren't exactly fixtures they are mocks

Reply:
How do you suggest using mock objects without fixtures? Do we want to create a new mock object for each test? 

> These aren't exactly fixtures they are mocks

I don't think the purpose of fixtures exclude using mock objects i.e. fixtures can be mocks.